src/pods/test-sentences/components/span.component.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { Typography } from '@material-ui/core';
+import { SentenceEntityVm } from '../test.sentences.vm';
+import * as styles from 'common/styles/tests.styles';
+
+interface Props {
+  sentenceSelected: SentenceEntityVm;
+  verbForms: string;
+  rightAnswerValue: boolean;
+}
+
+export const SpanComponent: React.FunctionComponent<Props> = props => {
```

Comment:
SpanComponent is too generic, let's use something more related to the domain, e.g.

SentenceComponent

Reply:
done